dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormField.vue

```diff
@@ -127,6 +129,16 @@ export default defineComponent({
   },
   methods: {
     selectNothingIfNotExistsFormKitPlugin,
+    /**
+     * Handle blur event on value input.
+     */
+    handleBlurValue() {
```

Comment:
I am not a big fan of this logic (appears elsewhere as well): It is my understanding, that this logic is meant to enforce: 1 - if there is a value for the datapoint ==> You need to select a quality, and 2 - if htere is no value for the datapoint ==> Auto-Select N/A.

However, when I input a value into the datapoint, then I can still select N/A as quality. This does not make sense. Additionally, I can set the datapoint to an empty string, and then manually re-select a quality different from N/A.

IMO we should either make this strict i.e., quality N/A if and only if there is no value in the value field, or give full control to the user.

Reply:
It is now limited. If there is something typed in the input value field, then NA will not be shown as an option.